spec/models/user_spec.rb

```diff
@@ -20,6 +20,40 @@
 require 'rails_helper'
 
 RSpec.describe User do
+  describe '::invitee_select_for' do
```

Comment:
when handling testing scopes (and naming the blocks) it should either be something like `context 'scopes'` then just a bunch of `describe '.invitee_select_for'`. You can also just do the describe alone. When a method is like `describe '.invitee_select_for'` the `.` implies it is a class method.